runtime/gc_modron_startup/mgcalloc.cpp

```diff
@@ -469,6 +469,9 @@ J9AllocateObject(J9VMThread *vmThread, J9Class *clazz, uintptr_t allocateFlags)
 	}	
 #endif /* J9VM_GC_THREAD_LOCAL_HEAP */	
 
+	if (NULL != objectPtr && J9_ARE_ALL_BITS_SET(clazz->classFlags, J9ClassContainsUnflattenedFlattenables)) {
+		vmThread->javaVM->internalVMFunctions->defaultValueWithUnflattenedFlattenables(vmThread, clazz, objectPtr);
+	}
```

Comment:
Should the same logic be applied in `J9AllocateObjectNoGC` as well?
Also there are two helpers for Indexable Object allocation `J9AllocateIndexableObjectNoGC` and `J9AllocateIndexableObject`. These helpers should be modified as soon as support for Unflattened Flattenables arrays will be implemented

Reply:
Also this code should be moved before call of `env->disableInlineTLHAllocate()` line 465

Reply:
@dmitripivkine Changes made. Please review, thank you! `J9AllocateIndexableObjectNoGC` and `J9AllocateIndexableObject` is going to be modified soon in my next PR.